index.html

```diff
@@ -77,17 +83,31 @@ <h3>IMSC1.0.1 Compatibility</h3>
       presented by an [[!ttml-imsc1.0.1]] presentation processor.</p>
 
       <p>The <code>#fillLineGap</code> and <code>#activeArea</code> specified in [[!ttml-imsc1.0.1]] SHALL be
-      <code>permitted</code> in [[!ttml-imsc1.1]].</p>
+      permitted in [[!ttml-imsc1.1]].</p>
     </section>
 
     <section>
       <h3>TTML2 Subset</h3>
 
-      <p>Notwithstanding deprecated syntax, as specified below, a valid [[!ttml-imsc1.1]] Document SHALL be a valid [[!TTML2]]
-      document.</p>
+      <p>A valid [[!ttml-imsc1.1]] Document SHALL be a valid [[!TTML2]] document.</p>
+
+      <p>Any [[!ttml-imsc1.0.1]] extension that (i) is not one of the following and (ii) has an equivalent feature in [[!TTML2]]
+      SHALL be <a>deprecated</a> in [[!ttml-imsc1.1]] and the corresponding [[!TTML2]] feature permitted:</p>
+
+      <ul>
+        <li><code>#activeArea</code></li>
+
+        <li><code>#fillLineGap</code></li>
+
+        <li><code>#forcedDisplay</code></li>
+
+        <li><code>#linePadding</code></li>
 
-      <p>[[!ttml-imsc1.0.1]] syntax that is not supported in [[!TTML2]] SHALL be mapped to an equivalent [[!TTML2]] syntax and
-      deprecated in [[!ttml-imsc1.1]].</p>
+        <li><code>#multiRowAlign</code></li>
+      </ul>
+
+      <p>When [[!ttml-imsc1.0.1]] syntax is <a>deprecated</a> and mapped to equivalent [[!TTML2]] syntax, both syntaxes SHALL NOT
```

Comment:
The paragraph immediately preceding the bullet list talks about extensions being deprecated and features being permitted, but this sentence talks about syntax. For consistency, change this to "When an IMSC 1.0.1 extension is deprecated and an equivalent TTML2 feature is permitted, a document instance shall not use both the deprecated extension and the equivalent TTML2 feature." (happy to take editorial tweaks - just making the point that it's about features and extensions not syntax now)

Reply:
Modified as suggested